test/qunit/model/FHIRFilterOperatorUtils.unit.js

```diff
@@ -0,0 +1,69 @@
+sap.ui.define([
+	"sap/fhir/model/r4/FHIRFilterOperator",
+	"sap/fhir/model/r4/FHIRFilterOperatorUtils",
+	"sap/fhir/model/r4/FHIRFilter"
+], function (FHIRFilterOperator, FHIRFilterOperatorUtils, FHIRFilter) {
+	"use strict";
+
+	QUnit.module("Unit-Tests: FHIRFilterOperatorUtils", {});
+
+	QUnit.test("Test function getFHIRFilterPrefix ", function (assert) {
+		var oFilter = new FHIRFilter({ operator: FHIRFilterOperator.EQ });
```

Comment:
could you format the test to more visible logical blocks e.g. inserting empty lines

Reply:
done